app/models/user.rb

```diff
@@ -111,28 +142,43 @@ def role_name
   def reindex_jurisdiction_user_size
     return unless jurisdiction.present?
 
-    jurisdiction.reindex if saved_change_to_jurisdiction_id? || saved_change_to_role? || destroyed? || new_record?
+    if saved_change_to_jurisdiction_id? || saved_change_to_role? ||
+         destroyed? || new_record?
+      jurisdiction.reindex
+    end
   end
 
   def refresh_search_index
     User.search_index.refresh
   end
 
   def confirmed_user_has_fields
-    errors.add(:user, "Confirmed user must have username") unless !confirmed? || username.present?
-    errors.add(:user, "Confirmed user must have first_name") unless !confirmed? || first_name.present?
-    errors.add(:user, "Confirmed user must have last_name") unless !confirmed? || last_name.present?
+    unless !confirmed? || username.present?
+      errors.add(:user, "Confirmed user must have username")
+    end
+    unless !confirmed? || first_name.present?
+      errors.add(:user, "Confirmed user must have first_name")
+    end
+    unless !confirmed? || last_name.present?
+      errors.add(:user, "Confirmed user must have last_name")
+    end
```

Comment:
my autoformat is interfering with this suggestion, and so I will leave it

Reply:
weird seems like the auto-formatters are out of sync. It should auto-format to the same on pre commit but looks like your auto-formatter is doing something extra. What formatter are you using? Should probably turn it off for this project since it will cause unnecessary PR complexity.

Reply:
Ruby LSP with syntax_tree

![image](https://github.com/bcgov/HOUS-permit-portal/assets/19396915/f0ef3ca0-bda0-4b1d-8500-0f72fb960227)